doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -396,6 +398,61 @@ Customize this intermediate local directory with the ``RAY_AIR_LOCAL_CACHE_DIR``
 
     ...
 
+.. _train-working-directory:
+
+Keep the original current working directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To disable the default behavior of Ray Train changing the current working directory,
+set the ``RAY_CHDIR_TO_TRIAL_DIR=0`` environment variable.
+
+This is useful if you want your training workers to access relative paths from the
+directory you launched the training script from.
+
+.. tip::
+
+    When running in a distributed cluster, you will need to make sure that all workers
+    have a mirrored working directory to access the same relative paths.
+
+    One way to achieve this is setting the
+    :ref:`working directory in the Ray runtime environment <workflow-local-files>`.
+
+.. testcode::
+
+    import os
+
+    import ray
+    import ray.train
+    from ray.train.data_parallel_trainer import DataParallelTrainer
+
+    os.environ["RAY_CHDIR_TO_TRIAL_DIR"] = "0"
+
+    # Write some file in the current working directory
+    with open("./data.txt", "w") as f:
+        f.write("some data")
+
+    # Set the working directory in the Ray runtime environment
+    ray.init(runtime_env={"working_dir": "."})
+
+    def train_fn_per_worker(config):
+        # Check that each worker can access the working directory
+        # NOTE: The working directory is copied to each worker and is read only.
+        assert os.path.exists("./data.txt"), os.getcwd()
+
+        # Write artifacts that you want to persist in the trial directory.
+        print(ray.train.get_context().get_trial_dir())
+
+    trainer = DataParallelTrainer(
+        train_fn_per_worker,
+        scaling_config=ray.train.ScalingConfig(num_workers=2),
+        run_config=ray.train.RunConfig(
+            # storage_path=...,
+            sync_config=ray.train.SyncConfig(sync_artifacts=True),
```

Comment:
Should we mention here the caveat of setting this environment variable, which is that local artifacts won't actually be written/synced anymore?